pyladies_cz.py

```diff
@@ -65,6 +65,11 @@ def city(city_slug):
     past_meetups = [m for m in meetups if not m['current']]
     registration_meetups = [
         m for m in current_meetups if m.get('registration_status')=='running']
+
+    members = read_yaml('teams/' + city_slug + '.yml', default=())
+    team = [m for m in members if not m.get("alumni", False)]
```

Comment:
What about having a single for loop here with only one call of `m.get` for each member? I know it won't make that big difference but it might improve readability.

Reply:
Changed it to a for loop.